app/controllers/application_controller.rb

```diff
@@ -61,13 +62,15 @@ def deny_access
     (User.current.logged? || request.xhr?) ? render_403 : require_login
   end
 
+  def require_ssl?
+    SETTINGS[:require_ssl]
+  end
+
   def require_ssl
```

Comment:
atm, ```require_ssl``` serves as redirect to 'https' when the request is  is from 'http'.
The rest has moved to ```require_ssl?```